teachertool/src/components/HomeScreen.tsx

```diff
@@ -27,14 +32,14 @@ const Welcome: React.FC = () => {
     );
 };
 
-interface CardProps {
+interface IconCardProps {
```

Comment:
Just wondering if we should have some inheritance for these cards. Are there going to be any shared properties between all these different props that we want to extract? It may be unnecessary more work, but I figured I'd start a conversation about it. For example, it seems like all cards should have titles, which two out of the three do, and I'm wondering if we should extract that into a more generic class.